companion/src/constants.h

```diff
@@ -59,10 +59,10 @@ enum BoardEnum {
 #define CPN_MAX_KEYS                   32
 #define CPN_MAX_MOUSE_ANALOGS          2
 
-const char * const ARROW_LEFT = "\xE2\x86\x90";
-const char * const ARROW_UP = "\xE2\x86\x91";
-const char * const ARROW_RIGHT = "\xE2\x86\x92";
-const char * const ARROW_DOWN = "\xE2\x86\x93";
+const char * const ARROW_LEFT = "\u2190";
+const char * const ARROW_UP = "\u2191";
+const char * const ARROW_RIGHT = "\u2192";
+const char * const ARROW_DOWN = "\u2193";
```

Comment:
WTF... not very "Uni" now is it?  CP1252?  I guess we could add some define check... those arrows are much nicer than the old ones.  I'll do a search...   

Besides the warnings, does the display the arrows on the trim buttons?